config/initializers/feature_flags.rb

```diff
@@ -0,0 +1,5 @@
+# Every time a content item is published, content-store updates Router's
+# backend address for that content item's rendering app.
+# If the environment variable NO_SET_ROUTER_BACKEND_ADDRESS_ON_PUBLISH is set,
+# content-store will not update backend addresses on publish.
+SET_ROUTER_BACKEND_ADDRESS_ON_PUBLISH = !ENV["NO_SET_ROUTER_BACKEND_ADDRESS_ON_PUBLISH"]
```

Comment:
There's a possible future source of confusion here where someone does:

```diff
- NO_SET_ROUTER_BACKEND_ADDRESS_ON_PUBLISH=true
+ NO_SET_ROUTER_BACKEND_ADDRESS_ON_PUBLISH=false
```

but `!"false"` is `false` in ruby.

Suggest:

```suggestion
SET_ROUTER_BACKEND_ADDRESS_ON_PUBLISH = ENV["NO_SET_ROUTER_BACKEND_ADDRESS_ON_PUBLISH"] != "true"
```

Reply:
Sure, no probs. I was going for "as simple and predictable as possible" but happy to go with your suggestion. (It still wouldn't handle "no" or "0", which are also quite commonly used and could reasonably be expected to work, but since it's probably only going to be used the once and only by Replatforming, I think that's fine. FWIW I'd usually go with `/^[1ty]/i.match?` but the prevailing style in this repo seemed to be `if ENV['foo']` hence I stuck with that style.)

Reply:
so some of the other boolean env vars in this codebase already have that potential issue, e.g. `MONGO_SSL=false` enables TLS on the `mongoid` client.

Reply:
oh wait no that's not a valid example, didn't notice it was ERB

Reply:
RAILS_LOG_TO_STDOUT and RAILS_SERVE_STATIC_FILES are example of that though.

Reply:
Yeah, your regex looks good / better - happy if you want to go for that.

I'm strongly of the view that testing for the presence of an env var is dangerous in general, because of the `VARIABLE=false` being evaluated to "yep, VARIABLE is non-empty, let's goooo". Anything that checks the actual value to some degree is a big improvement on that - `= "true"` is the simplest (but most naive) thing.

I wonder if I should put something in the style guide about this 🤔

Reply:
I agree — I think these things should follow the principle of least surprise! I'll go with the regex then 👍